rtv/subreddit.py

```diff
@@ -89,6 +91,42 @@ def open_link(self):
         global opened_links
         opened_links.add(url)
 
+    @SubredditController.register('p')
+    def post_submission(self):
+        # Abort if user isn't logged in
+        if not self.reddit.is_logged_in():
+            show_notification(self.stdscr, ['Login to reply'])
+            return
+
+        subreddit = self.reddit.get_subreddit(self.content.name)
+
+        # Make sure it is a valid subreddit for submission
+        # Strips the subreddit to just the name
+        sub = str(subreddit).split('/')[2]
+        if '+' in sub or sub == 'all' or sub == 'front':
+            message = 'Can\'t post to /r/{0}'.format(sub)
+            show_notification(self.stdscr, [message])
+            return
+
+        # Open the submission window
+        submission_info = SUBMISSION_FILE.format(name=sub)
+        curses.endwin()
+        submission_text = open_editor(submission_info)
+        curses.doupdate()
+
+        # Abort if there is no content
+        if not submission_text:
+            curses.flash()
+            return
+        try:
+            title, content = submission_text.split('\n', 1)
```

Comment:
Will raise ``ValueError: need more than 1 value to unpack`` if the text is only one line